### ISSUE TITLE
Appendix D

### ISSUE THREAD

**Issue by JSainath** [trevorlolsen/outline#6]
Hi Trevor, 

Am I right in assuming that D.1.1 is still work in progress? While you work on this, here are some things to keep in mind (I may add to this as I read more): 

(1) In D. 1. 1.2, first bullet point : 
I'd suggest you change "less and less efficient" to "less and less informative". Because "efficient" implies something about the size of the standard error in statistics. 

(2) D.1.2: It would be useful to illustrate the contents of this section with notation, similar to the discussion in BSM. Do you have the time to do that?

**Comment by trevorlolsen** [comment]
Yes, all of section D.1 is a work in progress. I'll keep this feedback in mind when I round it out. Thanks!
